packages/terra-image/examples/ImageVariant.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Image from '../src/Image';
+
+const ImageVariant = () => (
+  <div>
+    <h2>Default Image Style</h2>
+    <Image src="http://lorempixel.com/320/320/nature/" alt="example of defualt styles" />
+    <br />
```

Comment:
Maybe perhaps we can push the image in a paragraph instead of using a bunch of brs? One thing I found with blue_steel is that if you use patterns that are not recommended, then that behavior gets copied into the consumer applications very easily.